src/pytest_plugins/shared/execute_fill.py

```diff
@@ -81,6 +81,10 @@ def pytest_configure(config: pytest.Config):
         "markers",
         "execute: Markers to be added in execute mode only.",
     )
+    config.addinivalue_line(
+        "markers",
+        "zkvm: Tests that are relevant to zkVM.",
```

Comment:
@kevaundray in the context of EVM tests, I would have expected a `zkevm` marker. Any strong feelings or reason for `zkvm`? :slightly_smiling_face:  
```suggestion
        "zkevm: Tests that are relevant to zkEVM.",
```

Reply:
Hey :) zkEVM would be the more correct/specific term